api_reference.md

```diff
@@ -120,6 +122,27 @@ thoroughly.
 
 For more details on how these were trained, see the [Toxicity and sub-attribute annotation guidelines](https://github.com/conversationai/conversationai.github.io/blob/master/crowdsourcing_annotation_schemes/toxicity_with_subattributes.md).
 
+
+#### Toxicity models for Spanish, French and German comments
+
+The following models are our first non-English TOXICITY models. Only use these if you are
+interested in testing an experimental model and are willing to change your code once the
+production models are available. These models are experimental and have not been tested as
+thoroughly as their English counterparts.
+
+*   TOXICITY_EXPERIMENTAL
+*   SEVERE_TOXICITY_EXPERIMENTAL
+*   PROFANITY_EXPERIMENTAL
+*   IDENTITY_ATTACK_EXPERIMENTAL
+*   INSULT_EXPERIMENTAL
+*   THREAT_EXPERIMENTAL
+
+Please refer to the rest of the documentation bellow to appropriately set the
+language fields in your request.
+
```

Comment:
I think it's worth explicitly mentioning the difficulties of calling the API if the user doesn't know the language. Namely, if the user has data in both Spanish and English (for example), there's no way to make a single API call that automatically uses the right model depending on the language.

We can just say something like "Currently, if your comments are in both English and another experimentally-supported language (Spanish, French, German), there's no simple way to use the API that works for all comments unfortunately. You can either (1) change how you call the API depending on the comment language, or (2) wait until these models become "Alpha", when they'll be handled automatically."

Reply:
Great suggestion.